util/findLostPatchesOneRepository.sh

```diff
@@ -0,0 +1,121 @@
+#!/bin/sh
+
+###########################################
+#  
+#   Find any missing commits that are IN a given branch 
+#   but are NOT in the 2nd branch. 
+#  
+#   These missing commits are found via a concept
```

Comment:
You don't drive a Volvo and I'm pretty sure you don't have any piercings. So... quit your posturing, yo. :P